app/actions/navigation/index.js

```diff
@@ -346,6 +350,10 @@ export function mergeNavigationOptions(componentId, options) {
 
 export function showOverlay(name, passProps, options = {}) {
     const defaultOptions = {
+        layout: {
```

Comment:
@josephbaylon not sure why your are seeing that as this is the code to set the background transparent

Reply:
@josephbaylon if you confirm that you are on the right build and still see the notification issue (which should also apply on Android) then feel free to create another ticket. Do please confirm you are on the right build.

Reply:
@enahum I didn't reinstall and retested again and I can't reproduce either... maybe it's timing issue. I'll approve for now.